src/commands/release/import.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2016-2024 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Flags, Args } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { apply } from '@balena/release-bundle';
+import { createReadStream } from 'fs';
+import { ExpectedError } from '../../errors';
+
+export default class ReleaseImportCmd extends Command {
+	public static description = stripIndent`
+		Imports a release from a file to an application or fleet.
+
+		To save a release into a file, use 'balena release export'.
+
+		Use the --override-version option to specify the version 
+		of the imported release, overriding the one saved in the file.
+`;
+	public static examples = [
+		'$ balena release import ../path/to/release.tar -f 1234567',
+		'$ balena release import ../path/to/release.tar -f myFleet',
+		'$ balena release import ../path/to/release.tar -f myOrg/myFleet',
+		'$ balena release import ../path/to/release.tar -f myOrg/myFleet -V 1.2.3',
+	];
+
+	public static usage = 'release import <file>';
+
+	public static flags = {
+		fleet: { ...cf.fleet, exclusive: ['device'] },
```

Comment:
Flags should be for adjusting the default of a command if IMO and if I understand CLIs correctly. So `override-version` makes sense as a flag, but `fleet` does not. So the command should look like `$ balena release import ../path/to/release.tar 1234567`

Reply:
> Flags should be for adjusting the default of a command if IMO and if I understand CLIs correctly

I wouldn't mind if you could confirm my thinking on this @otaviojacobi 😅

Reply:
I agree with this - If it is mandatory, it shouldn't be a flag, but rather a mandatory arg.

Reply:
I changed it into the second argument instead.